build.py

```diff
@@ -213,7 +231,7 @@ def run_tests(race, parallel, timeout, no_vet):
             return False
     else:
         logging.info("Skipping 'go vet' call...")
-    test_command = "go test -v"
+    test_command = "go test"
```

Comment:
Might want to add this back, but under a flag (it annoyed me locally but is useful in CI to see that it's not frozen)

Reply:
I think we do want it back for CI, there I want all the info I can get after the fact